iotilecore/setup.py

```diff
@@ -64,7 +64,10 @@
             'tile_based = iotile.core.hw.virtual.tile_based_device:TileBasedVirtualDevice'
         ],
         'iotile.recipe_action': [
-            'FlashBoardStep = iotile.core.hw.debug.flash_board_step:FlashBoardStep'
+            'FlashBoardStep = iotile.core.hw.debug.flash_board_step:FlashBoardStep',
+            'ReflashTileStep = iotile.core.hw.debug.firmware_ship_steps:ReflashTileStep',
+            'SetFirmwareTagStep = iotile.core.hw.debug.firmware_ship_steps:SetFirmwareTagStep',
+            'SetUUIDStep = iotile.core.hw.debug.firmware_ship_steps:SetUUIDStep'
```

Comment:
We seem to already have the `SetUUIDStep` inside archtools that looks to be basically the same.  What's the advantage of duplicating it here vs fixing the other one if there's functionality missing?

https://code.eng.archsys.io/github.com/iotile/archtools/-/blob/archbuild/archbuild/recipe_steps/set_uuid_step.py

Reply:
Oh you are correct... I didn't realize it. I'll modify this one.